src/main/java/beam/physsim/jdeqsim/AgentSimToPhysSimPlanConverter.java

```diff
@@ -116,6 +117,58 @@ private void preparePhysSimForNewIteration() {
         jdeqsimPopulation = PopulationUtils.createPopulation(agentSimScenario.getConfig());
     }
 
+    /**
+     * Logs all the broken links (if any) observed with a path inside a network route.
```

Comment:
reports routes, which are not following road network

Reply:
done